contrib/cirrus/pr-should-link-jira.t

```diff
@@ -0,0 +1,144 @@
+#!/bin/bash
+#
+# tests for pr-should-link-jira.t
+#
+
+ME=$(basename $0)
+
+# Our test script queries github, for which we need token
+if [[ -z "$GITHUB_TOKEN" ]]; then
+    echo "$ME: Please set \$GITHUB_TOKEN" >&2
+    exit 1
+fi
+export CIRRUS_REPO_CLONE_TOKEN="$GITHUB_TOKEN"
+
+
+###############################################################################
+# BEGIN test cases
+#
+# Feel free to add as needed. Syntax is:
+#    <exit status> <commit message> <pr> # comments
+#
+# Where:
+#    exit status        is the expected exit status of the script
+#    commit message     commit message list
+#    dest branch        name of branch
+#    pr                 pr number
+#
+
+read -d '\n' msg_no_jira << EndOfText
+This is some text
+without a jira
+EndOfText
+
+read -d '\n' msg_invalid_jira << EndOfText
+This is some text
+without a jira
+Fixes #42
+More text...
+EndOfText
+
+read -d '\n' msg_jira << EndOfText
+This is some text
+with a jira
+Fixes https://issues.redhat.com/browse/RHEL-50507
+More text...
+EndOfText
+
+read -d '\n' msg_jira2 << EndOfText
+This is some text
+with a jira
+Fixes:  https://issues.redhat.com/browse/RHEL-50507
+More text...
+EndOfText
+
+read -d '\n' msg_multiple_jira << EndOfText
+This is some text
+with multiple jira lines
+Fixes  https://issues.redhat.com/browse/RHEL-50507
+More text...
+Fixes: https://issues.redhat.com/browse/RHEL-50506
+More text...
+EndOfText
+
+tests="
+0  8821 msg_no_jira main             not rhel branch, no link, should pass
+0  8821 msg_jira main                not rhel branch, link, should pass
+0  8821 msg_invalid_jira main        not rhel branch, invalid link, should pass
+0  8821 msg_no_jira v4.9             not rhel branch, no link, should pass
+1  8821 msg_no_jira v4.9-rhel        no link, no tag, should fail
+1  8821 msg_invalid_jira v4.9-rhel   invalid link, no tag, should fail
+0  8821 msg_jira v4.9-rhel           link, should work
+0  8821 msg_jira2 v4.9-rhel          link with colon, should work
+0  8821 msg_multiple_jira v4.9-rhel  multiple links, should work
```

Comment:
Could you table-align the branch names please? This is very difficult for a human to read

Reply:
done